releasenotes/notes/add-effective-dimension-class-44fd7661f6ba4b33.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Introduced two new classes, :class:`~qiskit_machine_learning.algorithms.EffectiveDimension`
```

Comment:
You should update the see algorithms \_\_init\_\_.py file to import these at this level and to have them in the docstring section too there so these new classes are linked into the API Ref. (Your unit test can then potentially import from here as we expect a user more commonly to do - rather than via the module itself as is currently done)